src/config.js

```diff
@@ -103,35 +99,42 @@ module.exports = {
         "To find out more, delve into our story on how we support people and organisations develop valuable apps and services accessible for everyone.",
       ],
       link: "/get-started/a-design-system",
-      linkText: "Read the story of our Design System",
+      buttonText: "Read our story",
     },
-    contribute: {
-      title: "Contribute",
+    design: {
+      title: "For designers",
       content: [
-        "The Design System is primarily built for the large numbers of apps and services we have.",
-        "While most of the work is undertaken by our Engineering teams, we've open sourced our Design System for the world to see, use and contribute to (which we think is very cool).",
+        "Whether you're building a new interface or refining an existing one, our Figma UI Kit, principles and documentation empowers you to create intuitive, consistent and accessible designs with ease.",
```

Comment:
I think it should be 'empower' here as it is multiple things that are empowering you.